### PR TITLE
Feature/pelagos 3369 manifest generator

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

**Comment by fightingtexasaggie** [comment]
Sample output data (and input) is on Jira the Jira ticket. Format looks a little different than William's example, but this might (or might not) be OK.

**Comment by praneethpr** [comment]
@fightingtexasaggie : The destination branch needs to be changed to develop

**Comment by fightingtexasaggie** [comment]
William is OK with the format, as he noted in https://griidc.atlassian.net/browse/PELAGOS-3348. Please review now.

File: share/python/directory_tree_node.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
+from __future__ import print_function
+import sys
+import os
+import platform
+
+'''
+@author: jvh
```
Comment (fightingtexasaggie) [inline share/python/directory_tree_node.py]: We generally don't put author in code comments. This is provided by github.

**Pull request merged**